test_programs/execution_success/regression_3635/src/main.nr

```diff
@@ -0,0 +1,10 @@
+// Test unsafe integer multiplication with overflow: 12^8 = 429 981 696
```

Comment:
Is this indeed the case here? Where is the multiplication mentioned here in the code below?

Reply:
No it is not, it is a remainder from a copy/paste. I just removed it, thank you.